tools/tasks/seed/app.through.ts

```diff
@@ -0,0 +1 @@
+import '../../../src/client/app/main';
```

Comment:
@mgechev it's used by `ng-xi18n` as an entry point to the application. Since `ng-xi18n` relies on the `tsconfig.json` and `src` folder it excluded, I needed to find a way in 🙄